modular_skyrat/modules/clock_cult/code/structures/traps/senders/pressure_sensor.dm

```diff
@@ -0,0 +1,45 @@
+/obj/item/clockwork/trap_placer/pressure_sensor
+	name = "pressure plate"
+	desc = "I wonder what happens if you step on it."
+	icon_state = "pressure_sensor"
+	result_path = /obj/structure/destructible/clockwork/trap/pressure_sensor
+
+/obj/structure/destructible/clockwork/trap/pressure_sensor
+	name = "pressure plate"
+	desc = "I wonder what happens if you step on it."
+	icon_state = "pressure_sensor"
+	unwrench_path = /obj/item/clockwork/trap_placer/pressure_sensor
+	component_datum = /datum/component/clockwork_trap/pressure_sensor
+	alpha = 60
+	layer = PRESSURE_PLATE_LAYER
```

Comment:
Note that this is exactly the same layer as `GAS_PUMP_LAYER` so if there's also eg a vent or scrubber on this tile, the layering between the two will be inconsistent.

Maybe adding code to the trap placer blocking this case should be added.

Reply:
I just axed pressure plate layer and made it sigil layer

Reply:
Awesome, that'll make it easier to maintain if Lemon changes the upstream layer values again.